### DISCUSSION TITLE
Best practice for styling content set by dangerouslySetInnerHTML

### DISCUSSION THREAD

**Discussion by DavidVaness** [ben-rogerson/twin.macro#468] [Q&A] [answered]
Hey,
 thank you for twin macro, it works really well in my dev workflow.
what I am wondering about is, how should I handle a situation like headless Wordpress that returns html.

I know there is a tailwindplugin like `@tailwindcss/typography` that could be used in this context but I would like to handroll some styles

Best regards 
David

**Comment by ben-rogerson** [comment, marked as answer]
hey David, great to hear that :)

Just like that typography plugin, you'll need to use a scoping class to style the children within the container.
It's probably best to keep this contained in a plugin so you could start with the [typography plugin](https://github.com/tailwindlabs/tailwindcss-typography/blob/master/src/index.js) and customise it.

**Reply by DavidVaness** [reply]
thank you :)
